modules/living_spaces_menus/living_spaces_menus.info.yml

```diff
@@ -13,3 +13,4 @@ default_content:
     - bfe5fcf7-5386-46ce-85b8-3bf9c392efe0
     - 4d89df5f-b766-4abc-a882-feeb218d61e2
     - 74a788e1-c016-4b94-8abc-0fc0b04612fd
+    - 67dd9262-c52f-4abd-a278-93c788e2dd55
```

Comment:
We also might add update hook to reimport missing menus, so the new menu is installed to AP/DIP clients